src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  width: container-width($square-size, 7, $sides-paddings);
```

Comment:
Please, place the styles in order as recommended [here](https://codeguide.academy/html-css.html#css-order)